packages/cozy-client/src/CozyClient.js

```diff
@@ -600,8 +600,17 @@ client.query(Q('io.cozy.bills'))`)
     return this.schema.validate(document)
   }
 
+  /**
+   * Creates or updates a document on the server
```

Comment:
nit: I think the infinitive form should be used, i.e. `Create or update a document on the server`